Fatras/include/ActsFatras/Digitization/detail/ParametersSmearer.hpp

```diff
@@ -0,0 +1,96 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <utility>
+
+#include "Acts/Utilities/Definitions.hpp"
+#include "Acts/Utilities/ParameterDefinitions.hpp"
+#include "Acts/Utilities/Result.hpp"
+
+namespace ActsFatras {
+
+/// Smearing functions definition:
+/// - it takes the unsmeared parameter
+/// - it returns the smeared parameter and a covariance
+using SmearFunction =
+    std::function<Acts::Result<std::pair<double, double>>(double)>;
+
+namespace detail {
+
+/// @brief single component smearing struct
```

Comment:
```suggestion
/// Single component smearing struct.
```
No `@brief` necessary.

Reply:
Removed.